src/plugins/kibana/public/settings/sections/indices/filebeat/directives/filebeat_wizard.html

```diff
@@ -31,19 +31,17 @@
 
   <div ng-switch="wizard.currentStep">
     <div ng-switch-when="0">
-      <paste-samples-step samples="wizard.stepResults.samples"></paste-samples-step>
+      <paste-samples-step samples="wizard.stepResults.sampleDocs"></paste-samples-step>
```

Comment:
This needs to be changed back to `samples`. `samples` is the unprocessed data from the user, wrapped in JSON for the pipeline creation step to consume. `sampleDocs` is the simulated output from the pipeline, which gets passed to the index pattern review step. My paste step PR #6005 adds another variable called `rawSamples` which is the untouched user input.

In other words:

`rawSamples` - user input for step 1
`samples` - JSON wrapped user input, output of step 1, input for step 2
`sampleDocs` - Simulated pipeline output, output of step 2, input for step 3